app-text/calibre/calibre-5.44.0-r1.ebuild

```diff
@@ -6,11 +6,15 @@ EAPI=8
 PYTHON_COMPAT=( python3_{9..11} )
 PYTHON_REQ_USE="ipv6(+),sqlite,ssl"
 
-inherit toolchain-funcs python-single-r1 qmake-utils xdg-utils
+inherit toolchain-funcs python-single-r1 qmake-utils verify-sig xdg-utils
 
 DESCRIPTION="Ebook management application"
 HOMEPAGE="https://calibre-ebook.com/"
-SRC_URI="https://download.calibre-ebook.com/${PV}/${P}.tar.xz"
+SRC_URI="
+	https://download.calibre-ebook.com/${PV}/${P}.tar.xz
+	verify-sig? ( https://calibre-ebook.com/signatures/${P}.tar.xz.sig )
+"
+VERIFY_SIG_OPENPGP_KEY_PATH="${BROOT}/usr/share/openpgp-keys/kovidgoyal.gpg"
```

Comment:
These changes appear to be safe to do without a revbump, but do we really need to modify 5.44.0-r1 when we're also creating 5.44.0-r2? If you're happy with consolidating all of your changes into the 5.44.0-r2 revbump, I have https://github.com/zmedico/gentoo/tree/calibre-5.44.0-r2 prepared to do that.

If you think it's really important to backport changes to 5.44.0-r1 then we can do that too, so let me know.

Reply:
I found it overall helpful to mentally structure the changes according to impact. I could hear arguments going both ways.

The main concrete differences are:
- some of those changes would result in faster merges without waiting for -r2
- although obviously the verify-sig bits have been confirmed at least once when committing it to the tree, so relying on Manifest hashes is "safe" by proxy as the binding signature on ::gentoo will cover the use of verify-sig... I figured some people might like to immediately take advantage of this by confirming it themselves and not waiting for -r2

If you would prefer the revbump I will defer to you.

Reply:
Okay, thanks for the explanation! I've merged all of your changes into 5.44.0-r2, and also added verify-sig and pkg_I changes to 5.44.0-r1 in 4103983d9ac27718dad9f5cbf9ea5bdf03aeb7ac and af5d2713d365e38de261c205cfd795edc337e540.